### ISSUE TITLE
Unable to send embeds in v1.3.0

### ISSUE THREAD

**Issue by GunnyWaffle** [replaceitem/carpet-discarpet#27]
I updated to v1.3.0 and found that I am unable to send messages with embeds.
```
Could not parse 'message_content' as 'message_content': Could not parse value in 'embeds' list with index 0: Cannot invoke "carpet.script.value.Value.getString()" because "this.image" is null in discord at line 268, pos 7
```
The code that fails: (where "embed" is any valid embed dict)
```
_embed(embed, channel) -> (
	task(_(e, c) -> (
		dc_send_message(c, {
			'content' -> '',
			'embeds' -> [e]
		});
	)
	, embed, channel);
);
```

**Comment by GunnyWaffle** [comment]
This bug persists in v1.3.1 for me.
note: I do not have the Scarpet Graphics mod.

**Comment by GunnyWaffle** [comment]
I did further testing and was able to send embeds if the `thumbnail` property was not used. This appears to be an issue where I provide a string URL.

**Comment by replaceitem** [comment]
Turns out it's just a copy&paste mistake 🤦‍♂️

**Comment by GunnyWaffle** [comment]
Hey, just a friendly poke to see how this is going. Just making sure, since you seemed to have identified the issue but no commit/release was made fixing it and this is marked closed still.

Thanks for the continued support. o/

**Comment by replaceitem** [comment]
I've fixed it in 10975c7015e387167a906c6106af9742079cedc5. I'm hoping to make a release soon too
